submissions/tosinIshola/HTML/Ecommerce.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>ALL THINGS FASHION</title>
+</head>
+<body>
+<header>
+    <h1><strong>ALL THINGS FASHION (A.S.F)</strong></h1>
+</header>
+
+<nav>
+    <div>
+        <h3>
```

Comment:
There isn't need for the `<div> and <h3>` on this block of code. The `<nav>` already serve that purpose